Figure S2

MikeLippincott opened the pull request:
Create Figure S1 - Platemaps

jenna-tomkinson commented on figures/S1.supplemental_figure1/figures/figureS1.png:
Some comments:

1. This is just a personal preference and opinion, but I am finding the blanks to be a bit jarring visually. Like there is already a lot going on with just PBMCs and SH-SY5Ys. My suggestion would be to remove the blank color and shape from the legend entirely and make those wells greyed out. It would make the figure just a bit less busy.
2. I would recommend adding titles over each of the different plates so that it is easier to understand that the top is the inducers and the bottom is the inhibitors. Currently, we only have the legends to know that.
3. I recommend removing one of the duplicate Cell Type legends.
4. I recommend making the inhibitor legend closer to the plate that it is associated with.

jenna-tomkinson commented on figures/S1.supplemental_figure1/figures/inducer_well_dose_platemap.png:
Some comments:

1. The plate map is insanely small versus the legend. Is this an intermediate file used to generate one figure or is this meant to be a stand-alone figure?
2. The Cell Type legend is missing the blank label.

jenna-tomkinson commented on figures/S1.supplemental_figure1/figures/inducer_well_platemap.png:
Some comments:

1. I am noticing that the shapes are off center in each of the wells. Recommend that these are in the center.
2. Recommend capitalizing the `blank` and `media` labels to match the rest.

jenna-tomkinson commented on figures/S1.supplemental_figure1/figures/inhibitor_well_platemap.png:
Some comments:

1. Same comment as previous plate map, the shapes are not centered in the wells.
2. Inhibitor legend is missing the `blank` value.

jenna-tomkinson commented on figures/S1.supplemental_figure1/figures/inhibtor_well_dose_platemap.png:
Some comments:

1. Same as the other file, the plate map is too small and the inhibitor legend is cut off at the bottom. Not sure if this is meant to be used in the figure or not.

jenna-tomkinson commented on figures/S1.supplemental_figure1/scripts/figure_S1.r:
```diff
@@ -0,0 +1,173 @@
+        )
+        # change the size of the shapes in the legend and plot
+    # 2 column legend
+    # + guides(shape = guide_legend(ncol = 2))
```
Recommend removing commented out code if not needed.

jenna-tomkinson reviewed (changes requested):
Great start to these plate maps! 🤩 I added some comments for you to address and I am requesting changes for the main figure due to duplicate legends and other comments. Please let me know if you want me to review again.

MikeLippincott commented:
I greyed out the blank wells and cleaned up the plate.
I added titles and cleaned up the legends. There was not a readily intuitive way to remove the cell type legend as platetools seems to be incompatible with aes overwriting... so I rearranged the cell type legend and changed shapes to make the figure less busy. Thank you!

jenna-tomkinson commented on figures/S2.supplemental_figure2/figures/figureS2.png:
LOVING the changes in this plot! This to be is much cleaner and very clear!

Some small comments:

1. In the inhibitor plate map, it is really hard to see the SH-SHY5Y shape in the dark blue wells.
2. Recommend either calling the wells with media either `Media ctr` or just `Media` to be consistent across platemaps.

jenna-tomkinson commented on figures/S2.supplemental_figure2/figures/inducer_well_platemap.png:
Just curious, are you planning on keeping these figures even though you have changed them for the figure?

jenna-tomkinson reviewed (approved):
The changes you made are wonderful, and these plate maps are very clear and tell the story of the experiment. Great job! 🎉
